plugins/core/src/main/kotlin/de/fayard/refreshVersions/core/extensions/gradle/Dependency.kt

```diff
@@ -23,5 +23,5 @@ internal fun Dependency.npmModuleId(): ModuleId.Npm {
 }
 
 internal fun Dependency.matches(moduleId: ModuleId): Boolean {
-    return moduleId.group == group && moduleId.name == name
```

Comment:
This leads to an allocation on every single `matches` check, that's the reason why I didn't just use the conversion trick.

Could you revert this change?

Reply:
No I can not, otherwise it does not work anymore, that was not just a cosmetic change:
```plain
moduleId.group == group && moduleId.name == name
actions == null && cache == @actions/cache
moduleId == moduleId()
Npm(group=actions, name=cache) == Npm(group=actions, name=cache)
```
I can try to reprogram it so that still the necessary calculation is done but no allocation necessary maybe.